CMakeTests.cmake

```diff
@@ -14,215 +14,213 @@
 
 # Test code for H3
 
-if(H3_IS_ROOT_PROJECT AND BUILD_TESTING)
-    option(BUILD_ALLOC_TESTS "Build tests for custom allocation functions" ON)
-    option(PRINT_TEST_FILES "Print which test files correspond to which tests" OFF)
+option(BUILD_ALLOC_TESTS "Build tests for custom allocation functions" ON)
+option(PRINT_TEST_FILES "Print which test files correspond to which tests" OFF)
+
+include(TestWrapValgrind)
```

Comment:
The visual diff here is lousy, but I assume from here on down it's just indentation changes?

Reply:
Correct, there should be no substantive changes to this file. Reindenting in VS Code and comparing to master I see only a couple lines where indents differ but no other changes.

Reply:
You can tell github diff to ignore indentation and then it clears up here.

Reply:
![Screenshot from 2022-05-25 19-48-58](https://user-images.githubusercontent.com/765551/170392686-84ba368b-62f7-4a38-82ec-f4f2442eb26f.png)
![Screenshot from 2022-05-25 19-49-11](https://user-images.githubusercontent.com/765551/170392688-c95fc5b0-8709-4813-9fb2-f0f06909a933.png)